lib/android-helpers.js

```diff
@@ -373,13 +373,23 @@ helpers.unlock = async function (adb) {
 
   await retryInterval(10, 1000, async () => {
     logger.debug("Screen is locked, trying to unlock");
-    await adb.startApp({
+
+    // first manually stop the unlock activity
+    await adb.forceStop('io.appium.unlock');
+
+    // then start the app twice, as once is flakey
+    let startOpts = {
       pkg: "io.appium.unlock",
       activity: ".Unlock",
       action: "android.intent.action.MAIN",
       category: "android.intent.category.LAUNCHER",
-      flags: "0x10200000"
-    });
+      flags: "0x10200000",
+      stopApp: false
+    };
+    await adb.startApp(startOpts);
+    await adb.startApp(startOpts);
```

Comment:
I wonder why starting it twice would make a difference. Is it something we could fix in the unlock_apk itself? (Like maybe running its own internal logic twice?)